resources/tsx/type.tsx

```diff
@@ -0,0 +1,13 @@
+import { string } from "yargs";
+type ObjectType = {
+    message: string;
+    name: string;
+    number: number;
+};
+type union = {a: 1} & {b: 2};
```

Comment:
Does that compile? Is `: 1` a type that represents the single number 1?

Reply:
Quite difficult. That are two object_types, too, I guess? But surrounded by whatever node is used for that intersection operator? I am not sure what would be the best approach to deal with that. But it is a quite exotic use case, probably.

Reply:
Further discussion about this topic is now under issue #213